Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -185,6 +264,60 @@ class StatusBar extends React.Component<{
    */
   static currentHeight = StatusBarManager.HEIGHT;
 
+  /**
+   * Get the current frame (height and y offset) of the Status Bar.
+   *
+   * Returns a Promise that resolves to a frame object with a `y` and `height`.
+   */
+  static getCurrentFrame(): Promise<{y: number, height: number}> {
```

Comment:
I think we have to name it `top`, not `x`.

Reply:
Good idea.